### PR TITLE
[ContactStructuralMechanicsApplication] Cleaning Parameters

### PR DESCRIPTION
**Pull request by loumalouomega**
**📝 Description**

Cleaning Parameters

**🆕 Changelog**

- [Cleaning utilities](https://github.com/KratosMultiphysics/Kratos/commit/ca51eae90f832c357c67579f9a1c53a7bdf69390)
- [Remove mesh_id](https://github.com/KratosMultiphysics/Kratos/commit/a75a9a76d59b589442715c9cbd83bbc7636b03a6)
- [More remove mesh_id](https://github.com/KratosMultiphysics/Kratos/commit/851b5cf60ee1d2f1ab587315e1c07530356bce41)
- [Reduce AddValue with CopyValuesFromExistingParameters](https://github.com/KratosMultiphysics/Kratos/commit/203de543f30ab6e996d4bb2909e04bb00fc07734)

### CODE REVIEW

File: applications/ContactStructuralMechanicsApplication/python_scripts/auxiliary_methods_solvers.py [hunk 1]
```diff
@@ -219,37 +219,12 @@ def  AuxiliaryCreateConvergenceParameters(main_model_part, settings, contact_set
...
-    conv_params.AddValue("mortar_type", contact_settings["mortar_type"])
-    conv_params.AddValue("condn_convergence_criterion", contact_settings["condn_convergence_criterion"])
-    conv_params.AddValue("print_convergence_criterion", contact_settings["print_convergence_criterion"])
-    conv_params.AddValue("ensure_contact", contact_settings["ensure_contact"])
-    conv_params.AddValue("frictional_decomposed", contact_settings["frictional_decomposed"])
-    conv_params.AddValue("compute_dynamic_factor", contact_settings["compute_dynamic_factor"])
-    conv_params.AddValue("gidio_debug", contact_settings["gidio_debug"])
+    
+    parameter_list = ["convergence_criterion", "rotation_dofs", "echo_level", "displacement_relative_tolerance", "displacement_absolute_tolerance", "residual_relative_tolerance", "residual_absolute_tolerance"]
+    conv_params.CopyValuesFromExistingParameters(settings, parameter_list)
```
Comment (AlejandroCornejo) [inline applications/ContactStructuralMechanicsApplication/python_scripts/auxiliary_methods_solvers.py:224]: very elegant!

**Review by AlejandroCornejo — APPROVED** [review]
ok, simple changes

**Comment by loumalouomega** [comment]
@AlejandroCornejo I needed to trigger CI

**Comment by loumalouomega** [comment]
> @AlejandroCornejo I needed to trigger CI

Looks like the approval is still there, nice

**Comment by philbucher** [comment]
> @AlejandroCornejo I needed to trigger CI

FYI you could manually rerun the CI, no need for another commit

**Pull request merged**